src/geometry/analytic_system.hpp

```diff
@@ -169,6 +169,10 @@ class Analytic {
     return system_.Lapse(X0, X1, X2, X3);
   }
   KOKKOS_INLINE_FUNCTION
+  Real Lapse(int b, Real X0, Real X1, Real X2, Real X3) const {
+    return system_.Lapse(X0, X1, X2, X3);
+  }
```

Comment:
Good idea. I'll add that.

Reply:
Actually... for analytic systems, there's no requirement `b==0`, because the underlying system presumably knows where it is. An analytic system might be called interchangeably with a cached system, and calls with `b != 0` should work in both cases.